Examples/iOS/Preference.swift

```diff
@@ -1,6 +1,6 @@
 struct Preference {
     static var defaultInstance = Preference()
 
-    var uri: String? = "rtmp://192.168.1.6/live"
-    var streamName: String? = "live"
+    var uri: String? = "rtmps://tempbalancer.eventlive.pro:443/live"
```

Comment:
There are some changes in this commit that are not related to the fix itself. Could you please remove it? This url, new DEVELOPMENT_TEAM, changing maxRetryCount, commenting code, new padding in stateDidChange. Thank you